manifests/mariadb.pp

```diff
@@ -64,14 +65,14 @@
     content => template('sunet/mariadb/cluster-status.erb.sh'),
     mode    => '0744',
   }
-  file { '/etc/sudoers.d/99-size-test':
+  file { '/etc/sudoers.d/99-cluster-size-test':
     ensure  => file,
     content => "script ALL=(root) NOPASSWD: /usr/local/bin/cluster-size\n",
```

Comment:
I must admit, I am not happy about the naming of these :/ should maybe be called `check_size`